includes/Handlers/Product/Woo_SOR.php

```diff
@@ -81,12 +81,76 @@ public static function update_catalog_item( CatalogObject $catalog_object, \WC_P
 			$item_data->setReportingCategory( $square_category );
 		}
 
-		$catalog_variations = $item_data->getVariations() ?: array();
+		$attributes = $product->get_attributes();
+
+		$product_variation_ids = $product->get_children();
+		$catalog_variations    = $item_data->getVariations() ? $item_data->getVariations() : array();
```

Comment:
Since PHP 7.0 we can change this to:
```suggestion
		$catalog_variations    = $item_data->getVariations() ?? array();
```